README.md

```diff
@@ -48,9 +48,9 @@ And in your solidity contract import it as:
 
 ## Build
 
-To get a list of all available `yarn` targets run the following inside `packages/contracts`:
+To get a list of all available `pnpm` targets run the following inside `packages/contracts`:
 
-    % yarn run info
+    % pnpm run info
```

Comment:
This command fails with the following error message.

```
Missing script: info
```

The equivalent to `yarn run info` seems to be just `pnpm run`. It prints:

```
Commands available via "pnpm run":
  build
    turbo run build
  dev
    turbo run dev --parallel
  lint
    turbo run lint
  format
    prettier --write "**/*.{ts,tsx,md}"
```

Reply:
It must be run from inside `packages/contracts`, as stated in the README

Reply:
Oops, you're right.